pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/BacklogQuota.java

```diff
@@ -28,6 +28,15 @@
  */
 public interface BacklogQuota {
 
+    /**
+     * Gets quota limit in size.
+     * Remains for compatible
+     *
+     * @return quota limit in bytes
+     */
+    @Deprecated
```

Comment:
I'd suggest to specify more the context here, maybe linking the PR and adding the "since" parameter to the @Deprecated annotation

Reply:
Added link and @since anootation

Reply:
@Shoothzj This result in not being added. Could you provide the context here?